docs/src/use/migrate-to-10.0.0.md

```diff
@@ -221,3 +222,30 @@ For example, this configuration is now invalid due to the duplicate `"u"` flag:
 **To address:** Remove any duplicate flags from your `allowConstructorFlags` array configuration of `no-invalid-regexp` rule. Each flag should appear only once in the array.
 
 **Related issue(s):** [#18755](https://github.com/eslint/eslint/issues/18755)
+
+## <a name="stricter-rule-tester"></a> Prohibiting `errors` or `output` of valid RuleTester test cases
```

Comment:
This should be after the end of the "Removal of deprecated `context` members" section, before the "Removal of `nodeType` property in `LintMessage` objects" section.